packages/cli/src/bin/index.ts

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env node
+import semver from 'semver';
+import chalk from 'chalk';
+import program from 'commander';
+import { init } from '../lib/init';
+import { log } from '../lib/log';
+import { board } from '../lib/board';
+import { start } from '../lib/start';
+import { devPlugin } from '../lib/devPlugin';
+import { dataset } from '../lib/dataset';
+import { serve } from '../lib/serve';
+import childProcess from 'child_process';
+
+const pkg = require('../package.json');
```

Comment:
You could enable the "resolveJsonModule" compiler option for this.

Reply:
This needs to change to import via enabling the `resolveJsonModule`.

Reply:
done.